roles/acm_spoke_mgmt/tasks/delete-ztp-by-ref.yaml

```
@@ -18,11 +18,11 @@
 - name: Delete ArgoCD Applications
   vars:
     app_sites: "{{ _asm_all_apps.resources | selectattr('spec.source.repoURL', 'equalto', asm_source_repo)
-                | selectattr('spec.source.path', 'equalto', 'sites')
+                | selectattr('spec.source.path', 'search', '(^|/)sites($)')
```

I believe sites must not have anything after, so its regex should be: `sites$`, right? using grouping makes it optional so it could match something like: `sites/something_else` or `sitesSomethingElse`
```suggestion
                | selectattr('spec.source.path', 'search', '(^|/)sites$')
```